app/controllers/app/items_controller.rb

```diff
@@ -0,0 +1,3 @@
+class App::ItemsController < App::BaseController
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 1 to 3